LearningLocker/learninglocker
# Moved the persona lock to after validation.

Closes https://github.com/LearningLocker/enterprise/issues/887

## To test
- Create a CSV import with MANY MANY rows of the same INVALID data for an existing persona
  - e.g. Ensure one set of actor data matches an existing persona's identifier
  - But additional idents should be invalid - e.g. a homepage that is an email, or an email column that isn't an email etc
- Spin up multiple workers
- Import and check that you get back validation errors
- Multiple workers should be able to process multiple jobs for the same "persona" simulataneosly
  - Previously, because the persona was locked before we validated the data, it meant only 1 worker could work on any single row of data at a time
  - Now, we validate before locking the persona, meaning we can process multiple rows at once
